Makefile

```diff
@@ -4,4 +4,4 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude lib
-	python3 -m unittest tests/*.py
+	pytest -v --cov app
```

Comment:
Can we have the HTML output locally as well as in Travis? Personally I find it useful for exploring areas that are lacking test coverage.

Reply:
will have a look